public_html/js/pinform.js

```diff
@@ -1406,4 +1526,40 @@ if (typeof PIN.Form.THIS_URL_PARAMS !== 'undefined') {
     }
 }
 
+PIN.Form.mpSortQuestions = function(queryData) {
```

Comment:
It's not clear to me what this method does. Is it re-sorting everything into "page" groups? Maybe just do that in the `sortQuestions` method instead?